CHANGELOG.md

```diff
@@ -1844,42 +1861,40 @@ rollback to 3.10.0 due to bug
 - Prompt users to re-agree to the Terms of Service when they are updated.
 
 ## [2.13.2] - 2016-10-04
-
+### Uncategorized
 - Fix bug where chosen FIAT exchange rate does no persist when switching networks
 - Fix additional parameters that made MetaMask sometimes receive errors from Parity.
 - Fix bug where invalid transactions would still open the MetaMask popup.
 - Removed hex prefix from private key export, to increase compatibility with Geth, MyEtherWallet, and Jaxx.
 
 ## [2.13.1] - 2016-09-23
-
+### Uncategorized
 - Fix a bug with estimating gas on Parity
 - Show loading indication when selecting ShapeShift as purchasing method.
 
 ## [2.13.0] - 2016-09-18
-
+### Uncategorized
 - Add Parity compatibility, fixing Geth dependency issues.
-- Add a link to the transaction in history that goes to https://metamask.github.io/eth-tx-viz
-  too help visualize transactions and to where they are going.
+- Add a link to the transaction in history that goes to https://metamask.github.io/eth-tx-viz to help visualize transactions and to where they are going.
```

Comment:
I moved this onto one line to make the formatting more consistent and easier to parse. I also fixed a typo here (`too` => `to`).